packages/itwinui-css/src/tile/tile.scss

```
@@ -237,7 +237,7 @@ $thumbnail-selector: ':is(.iui-thumbnail-icon, .iui-tile-thumbnail-picture)';
   margin-bottom: var(--iui-size-s);
   font-size: var(--iui-font-size-1);
   height: 100%;
-  max-height: calc(var(--iui-size-s) * 6);
+  max-height: 4.5em;
```

`max-height` changed to `em`.

1.5 line height * 3 desired lines of text = 4.5em.

---

Would `lh`/`rlh` be useful? https://css-tricks.com/lh-and-rlh-units/

Can use it with fallback for unsupported browsers:
```suggestion
  max-height: 4.5em;
  max-height: 3lh;
```
https://caniuse.com/mdn-css_types_length_lh

---

Ooh, either I didn't know of or forgot about `lh`!  Added in 2 places within `tile.scss`.